deploy/apache/sites-available/privacyidea.conf

```diff
@@ -11,6 +11,14 @@
 		AllowOverride None
 	</Directory>
 
+        # Yubico servers use /wsapi/2.0/verify as the path in the
+        # validation URL. Some tools (e.g. Kolab 2fa) let the 
+        # user/admin change the api host, but not the rest of
+        # the URL. Let's redirect the api URL to privacyideas
+        # /ttype/yubikey.
```

Comment:
Cornelius Kölbel <notifications@github.com> writes:

>> @@ -11,6 +11,14 @@
>>  		AllowOverride None
>>  	</Directory>
>>  
>> +        # Yubico servers use /wsapi/2.0/verify as the path in the
>> +        # validation URL. Some tools (e.g. Kolab 2fa) let the 
>> +        # user/admin change the api host, but not the rest of
>> +        # the URL. Let's redirect the api URL to privacyideas
>> +        # /ttype/yubikey.
>
> No, no redirect, please. Wondering why nginx config does that...

That's what I have now:

        # Yubico servers use /wsapi/2.0/verify as the path in the
        # validation URL. Some tools (e.g. Kolab 2fa) let the
        # user/admin change the api host, but not the rest of
        # the URL. Uncomment the following two lines to reroute
        # the api URL internally to privacyideas /ttype/yubikey.
        #RewriteEngine  on
        #RewriteRule    "^/wsapi/2.0/verify"  "/ttype/yubikey" [PT]

The PT flag is documented here:
https://httpd.apache.org/docs/current/rewrite/flags.html#flag_pt

If I call /wsapi/2.0/verify in my browser, the url is not rewritten,
e.g. unchanged after the call. access.log has:

 - - [22/Apr/2016:23:15:10 +0200] 200 "GET /ttype/yubikey HTTP/1.1"  167 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:45.0) Gecko/20100101 Firefox/45.0"

Do you have something different in mind?

Jochen

-- 
The only problem with troubleshooting is that the trouble shoots back.